mlflow/utils/nfs_on_spark.py

```diff
@@ -27,18 +28,23 @@ def get_nfs_cache_root_dir():
             spark_sess.conf.get("spark.databricks.mlflow.nfs.enabled", "true").lower() == "true"
         )
         if nfs_enabled:
-            nfs_root_dir = "/local_disk0/.ephemeral_nfs"
-            # Test whether the NFS directory is writable.
-            test_path = os.path.join(nfs_root_dir, uuid.uuid4().hex)
             try:
-                os.makedirs(test_path)
-                return nfs_root_dir
+                # If databricks runtime has dbutils entrypoint function `getReplNFSTempDir`,
+                # the directory it returns always has available permission.
```

Comment:
What is "available permission"?

Reply:
The returned directory is ensured to be read / write

Reply:
If so, please say so.